test/core/assert.cpp

```diff
@@ -12,9 +12,10 @@
 // http://www.boost.org/LICENSE_1_0.txt)
 
 
+#define BOOST_GEOMETRY_ENABLE_ASSERT_HANDLER
```

Comment:
This was caused by the PR for the define, which changed some code in `geometry_test_common.hpp`
Therefore the assert handler should be enabled earlier.